### PR TITLE
Added Relaunch FAQ

### PR DESCRIPTION
**Pull request by dficker**
Still needs some editing and polishing, for sure, but starting a FAQ for the Relaunch.

**Note:** Please fill out the PR Template to ensure proper processing.

Closes #

## Effect
PR includes the following changes:
- Added FAQ section for the Relanch Page
- Included a few questions that come up often with answers.

## Remaining Work
- [ ] List any outstanding work here
- [x] Technical review
- [ ] Copy Review

## Post Launch
**Do not remove** - To be completed by the docs team upon merge:
- [ ] Redirect `/docs/old-path/` => `/docs/new-path/` (if applicable)
- [ ] Include/exclude pages ^ respectively within docs search service provider (if applicable)
- [ ] Update Status Report
- [ ] Remove from the [project board](https://github.com/pantheon-systems/documentation/projects/14)

### CODE REVIEW

**Comment by alexfornuto** [comment]
Great stuff @dficker! If you're OK with my rewrites, I'll pass this on for final review.

**Comment by dficker** [comment]
Looks good o me! Thanks for the rewrites!

File: source/content/relaunch.md [hunk 1]
```diff
@@ -64,45 +66,46 @@ In order to retain Preferred Pricing an updated [invitation to pay](/add-client-
...
+DNS records propagate across many different servers and aren't refreshed until the record on *each server* up the tree expires. This means that a record with a 24 hour TTL can take several days to be updated across DNS servers globally. That's why we recommend lowering the TTL well before a site relaunch.
 
-It's recommended to lower the TTL well before a site relaunch. Remember that DNS records propagate across many different servers, and aren't refreshed until the record on *each server* up the tree expires. This means that a record with a 1-day TTL can take several days to be updated across DNS servers globally.
+Best practice during normal operation (e.g. not during a site relaunch) suggest a longer TTL (for example, 86400 seconds, or one day) because a long TTL helps reduce the number of DNS lookups that visitors' browsers need to perform. During a site relaunch, a long TTL can extend the time frame that return visitors are pointed to the old site, while new visitors are pointed to the new site.
```
[the remark this answers is not in the document]
Comment (carolyn-shannon) [reply source/content/relaunch.md]: practices

File: source/content/relaunch.md [hunk 2]
```diff
@@ -64,45 +66,46 @@ In order to retain Preferred Pricing an updated [invitation to pay](/add-client-
...
+1. In the new Site Dashboard, [standardize traffic for the primary domain](/domains/#redirect-to-https-and-the-primary-domain).
+1. In the old Site Dashboard, [downgrade the site from a paid plan to Sandbox](/site-plan/#cancel-current-plan).
+1. In the old Site Dashboard, [remove the existing card as a payment method for the site](/site-billing/#do-not-bill-this-site-to-a-card). If you're a contract customer, you can skip this step.
 
 ## Frequently Asked Questions
 ### Why is this process needed?
-While you can simply remove domains from an old site and then add them to a new one, HTTPS certificates will not be immediately available for the new site. This can cause security warnings for client browsers, and potentially affect processes that require a secure connection (like commerce transactions).
+HTTPS certificates will not be immediately available for the new site if, instead of following the steps above, you remove domains from the old site and add them to the new site. This can cause security warnings for client browsers, and potentially affect processes that require a secure connection (like commerce transactions).
```
[the remark this answers is not in the document]
Comment (carolyn-shannon) [reply source/content/relaunch.md]: modified this to follow if/then logic sequence

**Pull request merged**